update-imported-docs/README-hi.md

```diff
@@ -0,0 +1,63 @@
+
+# आयातित डॉक्स अपडेट करें
+
+यह स्क्रिप्ट डॉक्स फ़ाइलों को अपडेट करती है जो अन्य रिपॉजिट से उत्पन्न होती हैं।
+यह अपने इनपुट के रूप में एक YAML फ़ाइल नाम को स्वीकार करता है जिसे प्रति-रेपो पर अनुकूलित किया जा सकता है आधार।
+
+
+## आवश्यकताएँ
+
+आयातित डॉक्स को इन दिशानिर्देशों का पालन करना चाहिए:
+
+1.  [प्रलेखन शैली गाइड](/docs/home/contribute/style-guide/) का पालन करें।
+
+1. सामने के मामले में `शीर्षक` परिभाषित किया गया है। उदाहरण के लिए:
+    ```
+    ---
+    शीर्षक: सामग्री की तालिका में प्रदर्शित शीर्षक
+    ---
+
+    बाकी .md फ़ाइल ...
+    ```
+	
+1. `data` उपनिर्देशिका के तहत एक फ़ाइल में कहीं सूचीबद्ध किया जा सकता है, उदाहरण के लिए,
+   `data/imported.yml` फ़ाइल।
+
+1. सुनिश्चित करें कि `PyYAML` पैकेज स्थापित है:
+   
+```
+sudo apt-get install python-pip
+pip install PyYAML
+```
+
+## उपयोग
+
+इस निर्देशिका के भीतर, निम्नलिखित कमांड चलाएँ:
+
+```
++./update-imported-docs <CONFIG-FILE>
+```
+
+जहां `<CONFIG-FILE>` इस निर्देशिका में किसी भी YAML कॉन्फ़िगरेशन फ़ाइल हो सकती है।
+
+## कॉन्फ़िगरेशन फ़ाइल स्वरूप
+
+प्रत्येक कॉन्फ़िगरेशन फ़ाइल में एक से आयात किए जाने वाले कई रिपोज़ हो सकते हैं। जब आवश्यक हो, आप मैन्युअल रूप से संपादन करके कॉन्फ़िगरेशन फ़ाइल को अनुकूलित कर सकते हैं यह। आप दस्तावेज़ों के अन्य समूहों को आयात करने के लिए नई कॉन्फ़िग फ़ाइल बना सकते हैं। निम्नलिखित YAML कॉन्फ़िगरेशन फ़ाइल का एक उदाहरण है:
```

Comment:
1. instead of `रिपोज़ ` to `रेपोस` or english version `repos`.
2. replace `आयात` to `इम्पोर्टेड`
3. `संपादन ` can be replaced to `एडिटिंग`
4. `अनुकूलित` can be replaced to `कस्टमाइज्ड`